app/views/coronavirus_landing_page/components/shared/_section.html.erb

```diff
@@ -11,6 +11,9 @@
             href: item["url"],
             text: item["label"]
           }%>
+          <p class="govuk-body covid__list-item-description--featured">
+            <%= content_item_description item["url"] %>
```

Comment:
If this is blank, will it render an empty paragraph?

Reply:
good spot, was meant to do that but forgot